### PR TITLE
payment reference initial implementation

### PR DESCRIPTION
**Pull request by bodik**
pr created for code review by @pavlosherin 

please test at -dev site and comment

### CODE REVIEW

File: src/StagBundle/Entity/Participant.php [hunk 1]
```diff
@@ -72,7 +72,12 @@ class Participant {
 	 * @ORM\Column(name="payment", type="string", length=255, nullable=true)
 	 */
 	private $payment;
-	
+
+	/**
+	 * @ORM\Column(name="payment_reference", type="string", length=255, nullable=true)
```
Comment (pavlosherin) [inline src/StagBundle/Entity/Participant.php]: I think it should be only numeric of type to keep it consistent. E.g. FIO bank payment reference has /[0-9]{1,9}/ format.

Comment (bodik) [reply src/StagBundle/Entity/Participant.php]: i think it should stay string, we'll be able to support more payment methods in the furue

**Comment by bodik** [comment]
there should be name refactoring, payment reference clashes with symfony standards *Ref identifiers too much

should use paymentInvoiceId

File: src/StagBundle/Form/CourseType.php [hunk 2]
```diff
@@ -31,6 +31,7 @@ public function buildForm(FormBuilderInterface $builder, array $options) {
 		$builder->add('place', TextType::class, ["label" => "Místo"]);
 
 		$builder->add('color', TextType::class, ["label" => "Barva", "attr" => ["class" => "jscolor {hash:true, uppercase:false}"]]);
+		$builder->add('paymentInfo', TextType::class, ["label" => "Platební informace"]);
```
Comment (pavlosherin) [inline src/StagBundle/Form/CourseType.php]: should have require=false, if no payment info is added, than no payment info is send to participant

Comment (bodik) [reply src/StagBundle/Form/CourseType.php]: nice spot, thanx

Comment (bodik) [reply src/StagBundle/Form/CourseType.php]: fixed

**Pull request merged**